engine/src/main/java/org/hibernate/search/engine/service/impl/StandardServiceManager.java

```diff
@@ -221,7 +221,7 @@ synchronized void stopVirtual() {
 				if ( status != ServiceStatus.RUNNING ) {
 					stateExpectedFailure();
 				}
-				stopAndRemoveFromCache();
+				stopReal();
```

Comment:
If I understand correctly, this commit also make the wrappers remain in the cache when they are stopped. But I suppose it is on purpose?